### PR TITLE
[WIP] Dummy app on `now`

### PR DESCRIPTION
**Pull request by stevenwu**
###TODO

- [x] when we release a new tag, deploy server via travis (check npm release step)
- [x] make it so that the deployed server has a static domain name (so we don't have to modify the client's config when we deploy that; `*.now.sh` alias is fine)
- [x] when we release a new tag, also deploy client via travis; also with static `*.now.sh` alias
- [ ] add link to demo app on ember-simple-auth.com (after we verify this is working)

### CODE REVIEW

**Comment by stevenwu** [comment]
@marcoow where is the source code for ember-simple-auth.com?

**Comment by marcoow** [comment]
@stevenwu: in the [`gh-pages` branch](https://github.com/simplabs/ember-simple-auth/tree/gh-pages)

File: .travis.yml [hunk 1]
```diff
@@ -39,8 +39,17 @@ deploy:
     tags: true
     repo: simplabs/ember-simple-auth
 
+after_deploy:
+  - npm i -g now-pipeline
+  - now-pipeline-prune && now-pipeline --alias ember-simple-auth.now.sh
+  - cd server && now-pipeline-prune && now-pipeline --alias ember-simple-auth-server.now.sh
```
Comment (marcoow) [inline .travis.yml]: I think we need

```yaml
on:
  tags: true
  repo: simplabs/ember-simple-auth
```

here so we only ever deploy the latest tag (which is when we also release a new version to npm).

File: package.json [hunk 2]
```diff
@@ -17,6 +17,8 @@
     "diagrams": "mermaid --sequenceConfig guides/assets/esa-initial-flow.config.json --width 2400 -o guides/assets guides/assets/esa-initial-flow.txt",
     "fastboot": "echo \"Please also run npm start\"; ember fastboot --serve-assets",
     "lint": "eslint app addon blueprints config server test-support tests *.js",
+    "now-build": "ember build --environment=production",
```
Comment (marcoow) [inline package.json]: I wonder whether we should actually not build production so people can inspect the app's source easily. Or we could deploy sourcemaps as well.

File: package.json [hunk 3]
```diff
@@ -33,6 +35,8 @@
     "ember-cookies": "^0.0.13",
     "ember-fetch": "^2.1.0 || ^3.0.0",
     "ember-getowner-polyfill": "^1.1.0",
+    "fastboot-app-server": "^1.0.0-rc.5",
+    "node-fetch": "^1.7.1",
```
Comment (marcoow) [inline package.json]: Isn't this a subdependency of `ember-fetch` already?

File: .travis.yml (hunk 1 above)
Comment (stevenwu) [inline .travis.yml]: `after_deploy` should only run after deployments. We can keep an eye on Now and follow up if necessary.

**Comment by stevenwu** [comment]
This should be ready. I guess there's no way to verify results until next release?

**Comment by marcoow** [comment]
> I guess there's no way to verify results until next release?

I'm afraid not 😬 

> Prettier added some extra commas. Do we want to keep those?

let's only include related changes in this.

File: package.json (hunk 2 above)
Comment (marcoow) [inline package.json]: we do ship sourcemaps now

**Pull request merged**